### DISCUSSION TITLE
Public IP and projects

### DISCUSSION THREAD

**Discussion by Sesa1988** [supabase/supabase#8816] [General]
I want to query an API from two projects via database functions but they have rate limits per IP what is not a big deal but I have to questions here:

- Is the IP for one project static?
- Are multiple projects share the same IP?

**Comment by RWulff** [comment]
I would like to revive this conversation.

For my usecase it would be important to be able to have a restricted IP range. In particular I am trying to access another Postgres Database using the postgres_fdw extension to create foreign tables. But I can not whitelist 0.0.0.0 for security.

Ideally the public IP used would be static and not shared.
